recipes-app/node-red-gpio/files/0001-Dynamic-add-user-button-label.patch

```diff
@@ -0,0 +1,35 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Thu, 4 May 2023 10:31:09 +0800
+Subject: [PATCH] Dynamic add user button label
```

Comment:
"Dynamically"

Reply:
package bump to 0.4.0, which includes this patch already, so this patch will be removed.